Firestore/Example/Firestore.xcodeproj/project.pbxproj

```diff
@@ -624,6 +668,7 @@
 		6003F581195388D10070C39A = {
 			isa = PBXGroup;
 			children = (
+				618BBE7A20B89AAC00B5BCE7 /* cpp */,
```

Comment:
I called it CoreTestsProtos (since it's only used by the CoreTests).

(I couldn't figure out how to name it something other than the directory name via xcode, so I didn't realize it was possible. In the end, I edited the pbxproj file directly.)